yaml/ingress-inlets.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: expressjs-inlets-ingress
+  labels:
+    app: expressjs-inlets-ingress
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.global-static-ip-name: expressjs
+    kubernetes.io/tls-acme: "true"
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+    kubernetes.io/ingress.class: nginx
+  namespace: default
+spec:
+  rules:
+  - host: express.example.com
```

Comment:
Why did we need two entries / hosts?

Reply:
The www. is completely optional. But I thought maybe adding it would make sense if you have a CNAME record for your DNS name maybe?

Reply:
It maybe just me but it feels a classis www.mysite.com for people exposing their cluster for the first time 🤩

Reply:
Resolved!